lib/clients/metric-client/index.js

```diff
@@ -157,6 +148,25 @@ class MetricClient {
     _retry(retries, fn) {
         return fn().catch(err => (retries > 1 ? this._retry(retries - 1, fn) : Promise.reject(err)));
     }
+
+    _isEnabled() {
+        if (process.env.ASK_SHARE_USAGE === 'false') return false;
```

Comment:
adding this env variable to turn off telemetry for automated tests. we can document it for users too if we want.